src/mac/frame/header.rs

```diff
@@ -67,6 +68,34 @@ pub struct Header {
 
     /// Source Address
     pub source: Option<Address>,
+
+    /// Auxiliary security header, only Some if [`Header.security`] is true
+    pub auxiliary_security_header: Option<AuxiliarySecurityHeader>,
```

Comment:
Is this always going to be `Some`, if `security` is `true`. And `None`, if `security` is `false`? If so, can we just remove the `security` field?

Reply:
Yes, and yes, that does seem like the more logical approach. Fixed